src/xdp/xsk.c

```diff
@@ -2996,12 +2996,6 @@ XskSockoptSetUmem(
         goto Exit;
     }
 
-    //
-    // If support is needed for kernel mode AF_XDP sockets, UMEM MDL setup
-    // needs more thought.
-    //
-    ASSERT(RequestorMode == UserMode);
```

Comment:
There are no tests yet for verifying the kernel side of this yet, right?

Reply:
Right, this is all based on code review.